pkg/health/heartbeat/heartbeat.go

```diff
@@ -241,17 +242,37 @@ func UpdateDeltaSnapshotLease(ctx context.Context, logger *logrus.Entry, prevDel
 	}
 
 	renewedLease := deltaSnapLease.DeepCopy()
-	renewedLease.Spec.HolderIdentity = &revisionNumber
 	renewedTime := time.Now()
 	renewedLease.Spec.RenewTime = &metav1.MicroTime{Time: renewedTime}
 
+	//Update revision number only if revisions in existing lease is lower
```

Comment:
```suggestion
	//Update deltaSnapLease.Spec.HolderIdentity only when its value is less than latest deltaSnap.LastRevision
```
can you please update this comment with something similar or better.